.travis/script/gcc.sh

```diff
@@ -4,6 +4,7 @@ make &&
 make check &&
 # check display names
 if [[ -n ${ENABLE_UCS4+x} ]]; then
+    cd doc && make && cd .. &&
```

Comment:
What are you doing this for? afaik the info pages are built with `make all`. If you want the html for example you can just `make html` from the top level dir.

Reply:
Hmm are you sure they are built? I added this because I couldn't build the doc and I noticed a previous PR broke it. That must mean `make all` does not include the docs, right? Otherwise I would get an error when running `make`?